### DISCUSSION TITLE
google login not working in angular 14

### DISCUSSION THREAD

**Discussion by heyashwani** [abacritt/angularx-social-login#590] [Q&A]
{
    "error": "idpiframe_initialization_failed",
    "details": "You have created a new client application that uses libraries for user authentication or authorization that will soon be deprecated. New clients must use the new libraries instead; existing clients must also migrate before these libraries are deprecated. See the [Migration Guide](https://developers.google.com/identity/gsi/web/guides/gis-migration) for more information."
}

![image](https://user-images.githubusercontent.com/46573499/186212120-33fe4b15-8c64-44e3-ba32-1c8dd8afd427.png)

angular - 14
package used - angularx-social-login

**Comment by mspratt-biot** [comment]
You created your apps client id recently which requires the latest GIS APIs to be used. Make sure you've updated the social-login package to the latest. That should solve this problem.
